examples/server/server.cpp

```diff
@@ -3217,6 +3225,7 @@ int main(int argc, char **argv)
     sigemptyset (&sigint_action.sa_mask);
     sigint_action.sa_flags = 0;
     sigaction(SIGINT, &sigint_action, NULL);
+    sigaction(SIGUSR1, &sigint_action, NULL);
```

Comment:
This can be removed before we merge.  I was having trouble with some of my analysis tools with INT, so USR1 made it easier to iterate while testing the leak fix.